install.sh

```diff
@@ -1,4 +1,44 @@
 #!/usr/bin/env bash
+# bash-it installer
+show_usage() {
+  echo -e "\n$0 : Install bash-it"
+  echo -e "Usage:\n$0 [arguments] \n"
+  echo "Arguments:"
+  echo "--help (-h): Display this help message"
+  echo "--silent (-s): Install default settings without prompting for input";
+  echo "--interactive (-i): Interactively choose plugins"
+  exit 0;
+}
+
+echo "Installing bash-it"
```

Comment:
This line should probably be moved further down, to show only once the installation starts. At the moment it's also shown on the help screen.